webrtc/videoroom/assets/src/membraneWebRTC.ts

```diff
@@ -13,6 +13,8 @@ const phoenix_channel_push_result = async (push: Push): Promise<any> => {
 
 interface Participant {
   displayName: string;
+  mutedAudio: boolean;
+  turnedOffVideo: boolean;
```

Comment:
I would name it in the same way so maybe `mutedVideo`?

Reply:
Having varialbe named `mutedVideo` in place, where you want to mute audio in video element (`video.muted = true;`), IMO would be confusing